lib/transform.js

```diff
@@ -44,6 +44,7 @@ module.exports = class Transform {
                     slotMap.set(attribs.slot, [node]);
                 }
                 this._pushText(node.next, slotMap.get(attribs.slot));
+                delete attribs.slot;
```

Comment:
I'm not sure if copying all attributes besides slot would be faster. Would be interesting to see if attribs is a HashMap or V8 transforms it into a class...